homeassistant/components/tuya/switch.py

```diff
@@ -514,6 +514,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Curtain
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46o5mtfyc
+    "cl": (
+        SwitchEntityDescription(
+            key=DPCode.CONTROL_BACK,
+            name="Reverse Operation",
+            icon="mdi:restore",
```

Comment:
This is more fitting IMHO.

```suggestion
            icon="mdi:swap-horizontal",
```